.restyled.yaml

```diff
@@ -1,3 +1,7 @@
 restylers:
-  - brittany
+  - brittany:
+      include:
+        - "**/*.hs"
+        - "!library/Test/Hspec/Expectations/Json/Internal.hs" # CPP
```

Comment:
`!` under `include:` feels odd but that's neither here nor there.

Reply:
Yeah, it's `.gitignore`-inspired.